lib/bolt/transport/local/shell.rb

```diff
@@ -213,6 +223,15 @@ def execute(command, sudoable: true, **options)
           result_output.exit_code = t.value.exitstatus
           result_output
         end
+
+        def build_command(sudo_str, command_str, intput_stdin)
+          if intput_stdin
+            "#{sudo_str} sh -c 'echo #{@sudo_id} 1>&2; #{command_str}'"
```

Comment:
Why are these different for local vs ssh?

Reply:
Oops. I worked on the ssh transport first, then when I got to the local I realized I needed the string on stderr. I think they can be the same and I can move this to a shared method in `Sudoable::Connection`.